src/scss/components/_toolbar.scss

```diff
@@ -7,7 +7,7 @@
 
 .#{prefix(toolbar)} {
   display: flex;
-  align-items: flex-start;
+  align-items: center;
```

Comment:
have you checked the impact on mobile?

Reply:
This is the storybook from main:
![Screenshot from 2023-11-01 15-49-57](https://github.com/open-formulieren/open-forms-sdk/assets/19154114/2ebbe3d5-162a-4760-83e6-bbfa0b8a43b5)

This is from this branch:
![Screenshot from 2023-11-01 15-50-14](https://github.com/open-formulieren/open-forms-sdk/assets/19154114/e8eef251-2cb2-40e5-9923-da3cc6843107)

(Both look bad, but unrelated to this change)